models/health.go

```diff
@@ -74,53 +67,33 @@ type WorkloadStatus struct {
 
 // RequestHealth holds several stats about recent request errors
 type RequestHealth struct {
-	inboundErrorRate    float64
-	outboundErrorRate   float64
-	inboundRequestRate  float64
-	outboundRequestRate float64
-
-	ErrorRatio         float64 `json:"errorRatio"`
-	InboundErrorRatio  float64 `json:"inboundErrorRatio"`
-	OutboundErrorRatio float64 `json:"outboundErrorRatio"`
+	InboundRequestCode  map[string]map[string]float64 `json:"inboundRequestsCode"`
```

Comment:
We could simplify a little bit the naming here, changing `InboundRequestCode` to just `inbound`.
That would give us:

```json
{
  "requests": {
    "inbound": {
      "http": {
        "200": ...,
        "503": ...
      }
    },
    "outbound": {
      "http": {
        "200": ...
      }
    }
  }
}
```